src/test/java/org/opensearch/jobscheduler/rest/RestGetJobIndexActionTests.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.jobscheduler.rest;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.common.bytes.BytesArray;
+import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.jobscheduler.JobSchedulerPlugin;
+import org.opensearch.jobscheduler.model.JobDetails;
+import org.opensearch.rest.RestHandler;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.test.OpenSearchTestCase;
+import org.opensearch.test.rest.FakeRestChannel;
+import org.opensearch.test.rest.FakeRestRequest;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.List;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.Mockito.mock;
+
+public class RestGetJobIndexActionTests extends OpenSearchTestCase {
+
+    private RestGetJobIndexAction action;
+
+    public Map<String, JobDetails> jobDetailsHashMap;
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        jobDetailsHashMap = new HashMap<>();
+        action = new RestGetJobIndexAction(jobDetailsHashMap);
+    }
+
+    public void testGetNames() {
+        String name = action.getName();
+        Assert.assertEquals(action.GET_JOB_INDEX_ACTION, name);
+    }
+
+    public void testGetRoutes() {
+        List<RestHandler.Route> routes = action.routes();
+
+        Assert.assertEquals(
+            String.format(Locale.ROOT, "%s/%s", JobSchedulerPlugin.JS_BASE_URI, "_get/_job_index"),
```

Comment:
Make this a constant or at least instance variable. You also use it below in withPath()`.

Reply:
ACK